test_unstructured/chunking/test_base.py

```diff
@@ -572,24 +573,83 @@ class DescribePreChunkBuilder:
     def it_is_empty_on_construction(self):
         builder = PreChunkBuilder(opts=ChunkingOptions.new(max_characters=50))
 
-        assert builder.text_length == 0
-        assert builder.remaining_space == 50
+        assert builder._text_length == 0
+        assert builder._remaining_space == 50
```

Comment:
Interface-wise, these two attributes are replaced by `.will_fit(element)` and so become private.